docs/resources/iam_oidc_provider.md

```diff
@@ -10,7 +9,29 @@ description: |-
 
 Resource Type definition for AWS::IAM::OIDCProvider
 
-
+## Example Usage
+
+### Basic example
+Description about the first example
```

Comment:
should we update like: 

`To create IAM OpenID Connect provider with tags`

Reply:
updated the code.